data-raw/watershed/regional_approximation/description_regional_approx.R

```diff
@@ -0,0 +1,18 @@
+library(tidyverse)
+library(readxl)
+library(glue)
+
+print_regional_approx <- function(ws) {
```

Comment:
change function argument name to `watershed_name` from `ws` and then delete `watershed_name <- ws`

Reply:
done